deploy/examples/cluster.yaml

```diff
@@ -16,12 +16,12 @@ metadata:
 spec:
   cephVersion:
     # The container image used to launch the Ceph daemon pods (mon, mgr, osd, mds, rgw).
-    # v16 is Pacific, and v17 is Quincy.
+    # v17 is Quincy and v18 is Reef.
     # RECOMMENDATION: In production, use a specific version tag instead of the general v17 flag, which pulls the latest release and could result in different
-    # versions running within the cluster. See tags available at https://hub.docker.com/r/ceph/ceph/tags/.
-    # If you want to be more precise, you can always use a timestamp tag such quay.io/ceph/ceph:v17.2.6-20230410
+    # versions running within the cluster. See tags available at https://quay.io/repository/ceph/ceph?tab=tags.
+    # If you want to be more precise, you can always use a timestamp tag such quay.io/ceph/ceph:v18.1.3-20230726
     # This tag might not contain a new Ceph version, just security fixes from the underlying operating system, which will reduce vulnerabilities
-    image: quay.io/ceph/ceph:v17.2.6
+    image: quay.io/ceph/ceph:v18.1.3
```

Comment:
cluster.yaml is the example recommended for users in production. I'm thinking we don't want to set Reef as the production example until Rook v1.13. So if it's really just for the CI, we'll need to replace the version just during the CI.

Reply:
reverted